Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -38,15 +39,13 @@ public void OnInHandActivation(EntityUid user, DoorRemoteComponent component, Us
                     _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-toggle-bolts"), args.User, Filter.Entities(args.User));
                     break;
                 case DoorRemoteComponent.OperatingMode.ToggleBolts:
-                    component.Mode = DoorRemoteComponent.OperatingMode.OpenClose; // TODO: Swítch to ToggleEmergencyAcces when EA is implemented
-                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User)); // TODO: See the above comment
+                    component.Mode = DoorRemoteComponent.OperatingMode.ToggleEmergencyAccess;
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-toggle-emergency-access"), args.User, Filter.Entities(args.User));
                     break;
-            /*
                 case DoorRemoteComponent.OperatingMode.ToggleEmergencyAccess:
                     component.Mode = DoorRemoteComponent.OperatingMode.OpenClose;
                     _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User));
                     break;
-            */
             }
```

Comment:
Are these modes correct?

Reply:
The modes are correct, it determines the mode which to switch to by looking at the current mode.